benchmarks/benchmarks.py

```diff
@@ -75,3 +75,44 @@ def time_get_parts_python(self):
             parts.append(pygeos.get_geometry(self.multipolygons[i], range(num_parts)))
 
         parts = np.concatenate(parts)
+
+
+class STRtree:
+    """Benchmarks queries against STRtree"""
+
+    def setup(self):
+        self.polygons = pygeos.polygons(np.random.random((100, 3, 2)))
+        self.tree = pygeos.STRtree(self.polygons)
+
+    def time_tree_create(self):
+        pygeos.STRtree(self.polygons)
```

Comment:
It's fine to benchmark this, but we just need to be aware it's not the "tree" creation, but only the object construction (the tree is only created on first query)

Reply:
Right.  My goal was more to ensure that we didn't have a performance regression in the pygeos code to construct the tree from an array.  Perhaps it would be better to indicate this as `time_tree_insert`